lib/modules/manager/sbt/extract.ts

```diff
@@ -410,12 +407,12 @@ function prepareLoadPackageFiles(
       globalVars: {},
       packageFile,
     };
-    const res = extractPackageFile(content, acc);
+    const res = extractDependency(content, acc);
 
     if (res) {
       globalVars = { ...globalVars, ...res.localVars };
       if (res.registryUrls) {
-        registryUrls = [...registryUrls, ...new Set(res.registryUrls)];
+        registryUrls = [...registryUrls, ...res.registryUrls];
```

Comment:
see other comment.

Reply:
I moved to `extractAllPackageFiles` so it invokes new Set only 1 time before return final result